harvester/utils/ckan_utils.py

```diff
@@ -292,3 +293,7 @@ def ckanify_dcatus(metadata: dict, owner_org: str) -> dict:
     ckanified_metadata["extras"] = create_ckan_extras(metadata)
 
     return ckanified_metadata
+
+
+def add_uuid_to_package_name(name: str) -> str:
+    return name + "-" + str(uuid.uuid4())[:5]
```

Comment:
the ckan title/name munger limits the title to the length of `PACKAGE_NAME_MAX_LENGTH`. could adding the first 5 characters of a uuid str here cause the output to be longer than that? idk if ckan complains about a name being too long

Reply:
Ah good point. Let me take a look at the original code and if anything we shave those characters off the limit.

Reply:
I trimmed the `PACKAGE_NAME_MAX_LENGTH` to 90. Take a look at the changes and make sure they look good to you.